### PR TITLE
Clean up `feature/response-exception`

### PR DESCRIPTION
**Pull request by leofeyer**
Follow up on https://github.com/contao/contao-bundle/pull/5/commits

### CODE REVIEW

**Comment by aschempp** [comment]
Why did you remove all the docblocks?

**Comment by leofeyer** [comment]
They don't exists in Symfony either. They do exists in the interface, where the method is described, but not in any child class.

**Comment by aschempp** [comment]
That's true for one method where there is an interface, but you also removed it for the constructor and private variable. That means the IDE will not tell if `$content` is a string or anything else...

**Comment by leofeyer** [comment]
Don't know why they handle it this way, I only know they do: https://github.com/symfony/symfony/blob/master/src/Symfony/Component/HttpKernel/HttpKernel.php

**Comment by aschempp** [comment]
They do a lot of `{@inheritdoc}` which I think is unnecessary, but they did have a constructor argument list in your example?

**Comment by leofeyer** [comment]
True. I'll adjust my PR.

**Comment by leofeyer** [comment]
Adjusted in bf53764a3c46fa093d4b6d6b2dbd05e75e1bbf55. By the way, if we agree to add a PhpDoc comment to every method, I'm totally fine. We should just choose a convention and then stick to it.

> They do a lot of `{@inheritdoc}` which I think is unnecessary,

Me too.

**Comment by leofeyer** [comment]
I have updated my PR to reflect the necessary changes.

File: src/Exception/TemplateResponseException.php [hunk 1]
```diff
@@ -34,7 +39,7 @@ public function __construct($template, $statusCode = 200, array $headers = array
     }
 
     /**
-     * Return the template name
+     * Return the template path
```
Comment (aschempp) [inline src/Exception/TemplateResponseException.php]: It is not the path, it's just the name of the template.

Comment (leofeyer) [inline src/Exception/TemplateResponseException.php]: Fixed in ff6bb142bae4d138ca1baed419460f98a020b3b2.

**Pull request merged**